wiring/src/user.cpp

```diff
@@ -128,7 +128,7 @@ void serialEventRun()
 #endif
 }
 
-#if defined(STM32F2XX)
+#if defined(HAL_PLATFORM_BACKUP_RAM)
 #define PLATFORM_BACKUP_RAM 1
```

Comment:
Nit: ideally, we'd remove this if/else and define altogether and just use `HAL_PLATFORM_BACKUP_RAM` everywhere in place of `PLATFORM_BACKUP_RAM`.